GeeksCoreLibrary/Modules/GclReplacements/Interfaces/IStringReplacementsService.cs

```diff
@@ -120,6 +120,15 @@ public interface IStringReplacementsService
         /// <returns></returns>
         string EvaluateTemplate(string input);
 
+        /// <summary>
+        /// Searched input string for variables with default values and replaces them with those default values.
+        /// </summary>
+        /// <param name="input">The input string.</param>
+        /// <param name="prefix">The prefix that variables start with.</param>
+        /// <param name="suffix">The suffix that variables end with.</param>
+        /// <returns></returns>
+        string HandledVariablesDefaultValues(string input, string prefix = "{", string suffix = "}");
```

Comment:
Ik vind de benaming van deze functie misleidend. Het woord "`Handled`" duidt aan dat dit een functie is die bijhoudt welke "`DefaultValues`" al behandeld zijn. Ik denk dat de naam "`HandleVariablesDefaultValues`" beter is.